services/QuillLMS/engines/evidence/lib/evidence/evidence/open_ai/concerns/api.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module Evidence
+  module OpenAI
+    module Concerns::API
+      extend ActiveSupport::Concern
+
+      included do
+        include HTTParty
+        base_uri 'https://api.openai.com/v1'
+
+        attr_accessor :response
+      end
+
+      # classes using this concern require these methods
+      prepended do
```

Comment:
Interesting new syntax - I just read the docs. 

However, according to the [docs](https://ruby-doc.org/core-2.7.0/Module.html#method-i-prepend), prepended will only be called when the module is included via `prepend`, but we aren't using that directive anywhere in this PR or the codebase. Am I missing something?

Reply:
Ah, good catch, I was interpreting prepended incorrectly. You are right that it shouldn't be in this block. I've pulled it out and added a generic set of tests for the concern. Thanks for the review!